my-app/src/ui/public/Home.js

```diff
@@ -1,17 +1,26 @@
 import React from 'react';
+import { observer } from 'mobx-react';
 
 import { Input } from '../../shared/components/Input';
 import { VideoList } from '../../shared/components/VideoList';
 import videosStore from '../../shared/stores/VideosStore';
 import { VideoPlayer } from '../../shared/components/VideoPlayer';
-import { observer } from 'mobx-react';
+import { ViewedVideos } from '../../shared/components/ViewedVideos';
 
 export const Home = observer(() => {
   const onVideosInputChange = (e) => {
     videosStore.loadVideos(e.target.value);
   };
   const onListItemClick = (item) => {
     videosStore.currentVideoId = item.id.videoId;
+    videosStore.viewedList.push({
```

Comment:
you should create some function is videos store to update list, please never manipulate data from outside directly